packages/Search/test/CMakeLists.txt

```diff
@@ -11,11 +11,22 @@ TRIBITS_ADD_EXECUTABLE_AND_TEST(
   )
 TRIBITS_ADD_EXECUTABLE_AND_TEST(
   Exception
-  SOURCES tstException.cpp unit_test_main.cpp
+  SOURCES tstException.cpp
   COMM serial mpi
```

Comment:
@aprokop I overlooked that one when I reviewed #543 
I need to add `NUM_MPI_PROCS 1`.

Reply:
Done